src/event.cpp

```diff
@@ -298,6 +298,8 @@ void event::per_turn()
     g->add_zombie(eyebot);
     if (g->u.sees( place ))
      add_msg(m_warning, _("An eyebot swoops down nearby!"));
+    // One eyebot per trigger is enough, really
+    turn = int(calendar::turn);
```

Comment:
OK, this looks like it messes with the timer.  Gonna look at this when I have a chance to check that system out.